x-pack/plugins/security_solution/public/resolver/store/data/selectors.ts

```diff
@@ -301,3 +301,48 @@ export function databaseDocumentIDToAbort(state: DataState): string | null {
     return null;
   }
 }
+
+/**
+ * Return something? for a process.
+ * TODO, how does this work by taking entity id? what does it take when we want to use it w/ a legacy process event
+ */
+const relatedEventStatsForProcess: (
+  state: DataState
+) => (event: ResolverEvent) => ResolverNodeStats | null = createSelector(
+  relatedEventsStats,
+  (statsMap) => {
+    if (!statsMap) {
+      return () => null;
+    }
+    return (event: ResolverEvent) => {
+      const nodeStats = statsMap.get(uniquePidForProcess(event));
+      if (!nodeStats) {
+        return null;
+      }
+      return nodeStats;
+    };
+  }
+);
+
+/**
+ * Return the 'total' stats (whatever that means) for a process by entity ID
+ * TODO, how does this work for legacy events?
+ */
+export const relatedEventTotalForProcess: (
```

Comment:
What was the justification for this? I'd like to put it down in comments here.

Reply:
Found it in the other copy of the code: https://github.com/oatkiller/kibana/blob/resolver-refactoring/x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_related_counts.tsx#L54

Reply:
Now that I understand what this is doing I see that this number doesn't match the number of events.  Is there any point in showing this?

Reply:
@bkimmel you might want to comment here.

There is a difference between total events and the total amount of all the categories added up which is what is happening here. It really depends on what we want to communicate to the user. If we communicate that there are 5 total related events, when they look at the break down the values won't always added up to 5 because a single event could fall into multiple categories. So it could be confusing.

Reply:
For a set of related events with the following ECS categories:
`[DNS, File], [DNS, File], [Registry]`
We break counts in submenus as:
```
DNS: 2
File: 2
Registry: 1
```

There are 2 choices then, for how to display the total: `5` or `3`

If I'm not mistaken, I believe it was you, Robert, who suggested that `5` is less confusing (and I think everyone agreed). This is what `counts to 5`. There are implications in other places, but that's the crux of what's happening here.

Reply:
/shrug